apps/shopify/src/skuResolvers.js

```diff
@@ -143,11 +148,31 @@ export const fetchProductVariantPreviews = async (skus, config) => {
     body: JSON.stringify({ query }),
   });
 
-  const data = await res.json();
+  return await response.json();
+};
+
+/**
+ * Fetches the product variant previews for the product variants selected by the user.
+ */
+export const fetchProductVariantPreviews = async (skus, config) => {
+  if (!skus.length) {
+    return [];
+  }
+
+  const validIds = filterAndDecodeValidIds(skus, 'ProductVariant');
+
+  const requests = [];
+  for (let i = 0; i < validIds.length; i += SHOPIFY_ENTITY_LIMIT) {
+    response.push(
+      _fetchProductVariantPreviews(validIds.slice(i, i + (SHOPIFY_ENTITY_LIMIT - 1)), config)
+    );
+  }
+  const response = (await Promise.all(requests)).reduce(
```

Comment:
Where is `requests` being assigned? I see it's initialized above with an empty array, so wouldn't this be equivalent to `Promise.all([])`?

Reply:
Looks like you're on an older commit here. I accidentally pushed up that when refactoring to use Promise.all.  This is where the requests are being updated in the current code:

https://github.com/contentful/apps/pull/2944/files#diff-4a726df8342beeae0cec7ae3ccab3fed5adb3c4abeb148635af505d319365bd3R166